### PR TITLE
DOC: Update README

### PR DESCRIPTION
**Pull request by oesteban**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,44 +1,16 @@
-*fMRIPrep*: A Robust Preprocessing Pipeline for fMRI Data
+*fMRIPrep/rodents*: Analysis-grade BOLD fMRI data of rats
```
Comment (eilidhmacnicol) [inline README.rst:1]: I suppose once we get a mouse template, we can modify this to:
`Analysis-grade BOLD fMRI data of rats and mice` but this works until that time

File: README.rst [hunk 2]
```diff
@@ -1,44 +1,16 @@
...
-
-.. image:: https://img.shields.io/badge/RRID-SCR__016216-blue.svg
-  :target: https://doi.org/10.1038/s41592-018-0235-4
-  :alt: RRID:SCR_016216
+This tool adapts *fMRIPrep* to the use-case of rodent preclinical imaging -- 
+because MRI is some times done on species other than the *homo sapiens*.
 
 About
 -----
+Development was initiated by (soon-to-be) Dr. E. MacNicol in a research visit to the
```
Comment (eilidhmacnicol) [inline README.rst:8]: This is the first time I've seen this, and it is very surreal ☺️

File: README.rst [hunk 3]
```diff
@@ -100,7 +70,6 @@ Principles
    the process and decide which subjects should be kept for the group level
    analysis.
 
-
 Limitations and reasons not to use *fMRIPrep*
```
Comment (eilidhmacnicol) [inline README.rst:73]: Can maybe add that it is limited to the availability of non-human templates on templateflow? Of course with a reminder that we encourage others to add more non-human templates!

**Review by eilidhmacnicol — APPROVED** [review]
Looks good!

File: README.rst (hunk 2 above)
Comment (oesteban) [reply README.rst:8]: We'll remove the surrealism (and those parenthesis) pretty soon :)

File: README.rst (hunk 3 above)
Comment (oesteban) [reply README.rst:73]: sure, send suggestions! this was just a quick update

File: README.rst (hunk 1 above)
Comment (oesteban) [reply README.rst:1]: Okay I have stated that mice are planned to be supported, but not at the moment.

**Pull request merged**